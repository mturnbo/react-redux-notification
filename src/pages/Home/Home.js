import React from 'react';

const Home = () => (
  <div>
    <h2>Home</h2>
    <p>
      This single page app illustrates how to use Redux to manage notification messages displayed by a React
      component.  Messages can be displayed immediately or after a routing event.
    </p>
    <ul>
      <li>Click SHOW NOTIFICATION to immediately display notification</li>
      <li>
        Click SET NOTIFICATION to update notification message.  Then navigate to another page to see notification
        displayed after route change.
      </li>
    </ul>
  </div>
);

export default Home;
