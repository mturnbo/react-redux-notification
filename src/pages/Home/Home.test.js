import React from 'react';
import { mount } from 'enzyme';
import Home from './index';

const wrapper = mount(<Home />);

test('Home Page Renders', () => {
  expect(wrapper).toMatchSnapshot();
});
