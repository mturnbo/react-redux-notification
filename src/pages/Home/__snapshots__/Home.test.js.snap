// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Container Renders 1`] = `
<Home>
  <div>
    <h2>
      Home
    </h2>
    <p>
      Bacon ipsum dolor amet frankfurter hamburger picanha porchetta chuck. Doner salami burgdoggen ball tip, short loin meatloaf andouille pork belly picanha sirloin pancetta. Strip steak chicken drumstick jowl burgdoggen buffalo tongue cupim kevin capicola doner andouille bresaola ground round. Brisket cow jowl salami prosciutto short ribs fatback pork loin spare ribs burgdoggen turducken chuck.
    </p>
    <p>
      Turducken short ribs turkey biltong venison short loin ham hock bacon sirloin filet mignon pancetta beef. Strip steak tail tongue, short loin chicken shoulder short ribs buffalo. Burgdoggen ham hock turducken, picanha pork tail bresaola alcatra tongue bacon. Ribeye bresaola pancetta, filet mignon pork fatback prosciutto swine turducken shoulder capicola. Prosciutto biltong short ribs capicola spare ribs filet mignon pastrami. Burgdoggen biltong andouille alcatra brisket.
    </p>
  </div>
</Home>
`;

exports[`Home Page Renders 1`] = `
<Home>
  <div>
    <h2>
      Home
    </h2>
    <p>
      Bacon ipsum dolor amet frankfurter hamburger picanha porchetta chuck. Doner salami burgdoggen ball tip, short loin meatloaf andouille pork belly picanha sirloin pancetta. Strip steak chicken drumstick jowl burgdoggen buffalo tongue cupim kevin capicola doner andouille bresaola ground round. Brisket cow jowl salami prosciutto short ribs fatback pork loin spare ribs burgdoggen turducken chuck.
    </p>
    <p>
      Turducken short ribs turkey biltong venison short loin ham hock bacon sirloin filet mignon pancetta beef. Strip steak tail tongue, short loin chicken shoulder short ribs buffalo. Burgdoggen ham hock turducken, picanha pork tail bresaola alcatra tongue bacon. Ribeye bresaola pancetta, filet mignon pork fatback prosciutto swine turducken shoulder capicola. Prosciutto biltong short ribs capicola spare ribs filet mignon pastrami. Burgdoggen biltong andouille alcatra brisket.
    </p>
  </div>
</Home>
`;
